spec/integration/elasticsearch/transform/manifest.spec.yml

```diff
@@ -1,18 +1,45 @@
-##
-## Describes the specification for a transform's manifest.yml file
-##
-spec:
-  type: object
-  additionalProperties: false
-  definitions:
-    index_template:
-      $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template"
-  properties:
-    destination_index_template:
-      description: Elasticsearch index template for the transform's destination index
-      $ref: "#/definitions/index_template"
-      required: false
-    start:
-      description: Determines if the transform will be started upon installation
-      type: boolean
-      default: true
\ No newline at end of file
+type: object
+additionalProperties: false
+definitions:
+  index_template:
+    $ref: "../../data_stream/manifest.spec.yml#/properties/elasticsearch/properties/index_template"
+properties:
+  destination_index_template:
+    description: Elasticsearch index template for the transform's destination index
+    type: object
+    additionalProperties: false
+    properties:
+      settings:
+        type: object
+        additionalProperties: false
+        properties:
+          index:
+            type: object
+            additionalProperties: false
+            properties:
+              mode:
```

Comment:
We should probably add a patch so this is not used on packages with a format version older than 3.4. I will do this in a follow up.